Mimir/Repositories/DiffRepository.cs

```diff
@@ -0,0 +1,63 @@
+using Lib9c.GraphQL.Enums;
+using Mimir.Services;
+using MongoDB.Bson;
+using MongoDB.Driver;
+
+namespace Mimir.Repositories;
+
+public abstract class DiffRepository
+{
+    private readonly Dictionary<PlanetName, IMongoCollection<BsonDocument>> _collections = new();
+    private readonly Dictionary<PlanetName, IMongoDatabase> _databases = new();
+
+    protected DiffRepository(
+        MongoDBCollectionService mongoDbCollectionService,
+        string collectionName)
+    {
+        foreach (var planetName in new[] { PlanetName.Heimdall, PlanetName.Odin })
+        {
+            var databaseName = PlanetNameToDatabaseName(planetName);
+            var collection = mongoDbCollectionService.GetCollection<BsonDocument>(
+                collectionName,
+                databaseName);
+            _collections[planetName] = collection;
+
+            var database = mongoDbCollectionService.GetDatabase(databaseName);
+            _databases[planetName] = database;
+        }
+    }
+
+    private static string PlanetNameToDatabaseName(PlanetName planetName)
+    {
+        return planetName switch
+        {
+            PlanetName.Heimdall => "heimdall_diff_test",
```

Comment:
`_diff_test` collections are out dated

Reply:
So we don't need`DiffRepository` anymore

Reply:
Oh, thanks!